packages/suite/src/utils/wallet/coinmarket/coinmarketUtils.ts

```diff
@@ -273,3 +276,24 @@ export function processSellAndBuyQuotes<T extends CoinmarketTradeBuySellType>(
 
     return quotes;
 }
+
+export const getBestRatedQuote = (
+    quotes: CoinmarketTradeDetailType[] | undefined,
+    type: CoinmarketTradeType,
+): CoinmarketTradeDetailType | undefined => {
+    // best rate in buy or exchange according to receiveStringAmount, in sell according to fiatStringAmount
+    const getTypedAmount = (quote: CoinmarketTradeDetailType) => {
+        if (type === 'sell') {
+            return new BigNumber((quote as SellFiatTrade).fiatStringAmount ?? 0);
+        }
+
+        return new BigNumber((quote as BuyTrade | ExchangeTrade).receiveStringAmount ?? 0);
+    };
+
+    const bestRatedQuotes = quotes
+        ? quotes.toSorted((a, b) => getTypedAmount(b).minus(getTypedAmount(a)).toNumber())
```

Comment:
is `toSorted` supported in all browsers suite supports?

Reply:
`sort` will be safer! Thanks